tests/gnrc_ndp/Makefile

```diff
@@ -11,6 +11,8 @@ USEMODULE += netdev_test
 CFLAGS += -DGNRC_NETTYPE_NDP=GNRC_NETTYPE_TEST
 CFLAGS += -DTEST_SUITES
 
+INCLUDES += -I$(RIOTBASE)/sys/net/gnrc/pktbuf_static/include
```

Comment:
This is because we do not want to include the actual module?

Reply:
Huh? This is a module-internal include path. It is not added to the application's include path by the build system, so we need to add it manually.

Reply:
Ahh ok, my bad